src/index.html

```diff
@@ -11,7 +11,189 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      method="POST"
+      action="http://localhost:3000/api">
+
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+
+        <lable class="input-row">
+          Surname:
+
+          <input
+          type="text"
+          name="surnname"
+          autocomplete="off"
+          >
```

Comment:
proofread your code with checklist
```suggestion
          <input
            type="text"
            name="surnname"
            autocomplete="off"
          >
```

Reply:
fix all code style cases